### PR TITLE
update info.json file for keyboard

### PR DESCRIPTION
**Pull request by greenjack-z**
## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by greenjack-z** [comment]
this layout is in line with a physical rows at PCB, it was got from keyboard firmware builder generated files. That firmware works well, but after compiling under QMK Configurator - there are displacements/ You are right.
So which way is right?

**Comment by noroadsleft** [comment]
> this layout is in line with a physical rows at PCB, it was got from keyboard firmware builder generated files. That firmware works well, but after compiling under QMK Configurator - there are displacements/ You are right.
> So which way is right?

The idea of the layout macro ([this block of code](https://github.com/qmk/qmk_firmware/blob/ff91b86bcb16240b062bef6145f23519dde00d15/keyboards/handwired/zergo/zergo.h#L21-L35)) is that the bottom section represents how your switches are connected, and the top represents where they are geographically. If the 5 key is between 4 and 6 on the assembled keyboard, its switch identifier should be between those for 4 and 6 in the top section, regardless of what's going on electrically.

The [Clueboard 66 rev4](https://github.com/qmk/qmk_firmware/blob/5772ac8bf68f5e8e003b4a047494ac3fa2065524/keyboards/clueboard/66/rev4/rev4.h#L53-L70) has a good example of this. The 8, I, K and M keys are actually electrically in the same column of the switch matrix as the Esc, Tab, Caps Lock, Left Shift, and Left Ctrl keys.

**Comment by greenjack-z** [comment]
great example! thanks. I will fix it now.

**Comment by noroadsleft** [comment]
@greenjack-z You should flash your board from this branch and make sure the keys are assigned correctly.

If I'm right, your key assignments will be out of place, similarly to how the assignments looked in QMK Configurator.

**Comment by greenjack-z** [comment]
> You should flash your board from this branch
what is the best way to do that?

**Comment by noroadsleft** [comment]
I just realized you're probably using GitHub's web interface to make these changes. You would actually need a development environment set up to do it properly. You should look into [setting up a development environment](https://docs.qmk.fm/#/newbs_getting_started), but for now, don't worry about it. I checked and the keymap does need to be updated.

**Comment by greenjack-z** [comment]
> I just realized you're probably using GitHub's web interface to make these changes.
Sorry, but yes. It's not easy to set up environment and I do that in parallel, however the main issue is to be able to use the QMK Configurator without mistakes.

File: keyboards/handwired/zergo/info.json [hunk 1]
```diff
@@ -1,13 +1,14 @@
 {
     "keyboard_name": "zergo", 
-    "url": "", 
+    "url": "https://github.com/qmk/qmk_firmware/tree/master/keyboards/handwired/zergo", 
     "maintainer": "greenjack", 
     "width": 15, 
     "height": 6.25,
     "layouts": {
         "LAYOUT": {
+            "key_count": 80,
```
Comment (fauxpark) [inline keyboards/handwired/zergo/info.json:9]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by greenjack-z** [comment]
looks like not all the docs are up to date? cause some requirements are deprecated

**Review by fauxpark — APPROVED** [review]

File: keyboards/handwired/zergo/keymaps/default/keymap.c [hunk 2]
```diff
@@ -20,17 +20,17 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
...
-		KC_HOME, KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_PGUP, 
-		KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_5, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_BSLS, 
-		KC_LCTL, KC_A, KC_S, KC_D, KC_F, KC_G, KC_RBRC, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_LCTL, 
-		MT(MOD_LSFT, KC_END), KC_UP, KC_Z, KC_X, KC_C, KC_V, KC_ENT, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, MT(MOD_LSFT, KC_PGDN), 
-		KC_LEFT, KC_DOWN, KC_RGHT, KC_LALT, KC_SPC, KC_BSPC, KC_SPC, KC_RALT, KC_LGUI, KC_RGUI, LT(1, KC_NO)),
+		KC_DEL, KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSPC, 
+		KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, TG(1), KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_BSLS, 
+		KC_LCTL, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_RCTL, 
+		MT(MOD_LSFT, KC_HOME), KC_Z, KC_X, KC_C, KC_V, KC_B, KC_BSPC, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_RBRC, MT(MOD_RSFT, KC_PGUP), 
+		MT(MOD_LCTL, KC_END), KC_DOWN, KC_RGHT, KC_LALT, KC_SPC, KC_ENT, KC_SPC, KC_RALT, KC_LGUI, KC_MENU, MT(MOD_RCTL, KC_PGDN),
```
Comment (noroadsleft) [inline keyboards/handwired/zergo/keymaps/default/keymap.c:27]: Missing an end parenthesis here.

```suggestion
		MT(MOD_LCTL, KC_END), KC_DOWN, KC_RGHT, KC_LALT, KC_SPC, KC_ENT, KC_SPC, KC_RALT, KC_LGUI, KC_MENU, MT(MOD_RCTL, KC_PGDN)),
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**